app/styles/libs/_helpers.scss

```diff
@@ -103,6 +103,24 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
   white-space: nowrap;
 }
 
+.extra-link {
+  width: 49.5333px;
+  height: 46.3px;
+}
+
+.clamp {
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
```

Comment:
Not required

Reply:
Sir line-clamp do not work without vendor prefixes

Reply:
That means it's a very new feature. Think of other solution